File: seguridad/app/model/Usuarios.js
Ext.define('MyApp.model.Usuarios', {
    extend: 'Ext.data.Model',

    config: {
	    fields: [
	    { name:'UsuarioId', type:'string' },
	    { name:'UsuarioNombre', type:'string' },
	    { name:'UsuarioApellido', type:'string' },
	    { name:'UsuarioNombreCompleto', type:'string', useNull: true, defaultValue: null },
	    { name:'UsuarioLevel', type:'int' },
	    { name:'UsuarioPassword', type:'string' }
	    ]
	}
});

File: MullerWA/MullerWA/Models/Common/Login/Usuarios.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Web;


namespace MullerWA.Models
{
    public class Usuario
    {
        public string UsuarioId { get; set; }
        public string UsuarioNombre { get; set; }
        public string UsuarioApellido { get; set; }
        public string UsuarioNombreCompleto { get; set; }
        public int UsuarioLevel { get; set; }
        public string UsuarioPassword { get; set; }
    }
}